static/src/js/components/GranuleImage/__tests__/GranuleImage.test.js

```diff
@@ -1,70 +1,68 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
+import {
+  render,
+  screen
+} from '@testing-library/react'
+import { act } from 'react-dom/test-utils'
+import userEvent from '@testing-library/user-event'
+import '@testing-library/jest-dom'
 import GranuleImage from '../GranuleImage'
 
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup(overrideProps) {
-  const props = {
-    imageSrc: '',
-    ...overrideProps
-  }
-
-  const enzymeWrapper = shallow(<GranuleImage {...props} />)
-
-  return {
-    enzymeWrapper,
-    props
-  }
+const setup = (props) => {
+  act(() => {
+    render(
+      <GranuleImage imageSrc={props.imageSrc} />
+    )
+  })
 }
 
 describe('GranuleImage component', () => {
   describe('when no image is present', () => {
     test('renders itself correctly', () => {
-      const { enzymeWrapper } = setup()
-
-      expect(enzymeWrapper.type()).toBe(null)
+      const props = {
+        imageSrc: ''
+      }
+      setup(props)
+      expect(screen.queryByRole('button')).toBeNull()
     })
   })
 
   describe('when an image is present', () => {
     test('renders itself correctly', () => {
-      const { enzymeWrapper } = setup({
+      const props = {
         imageSrc: '/some/image/src'
-      })
-
-      expect(enzymeWrapper.type()).toBe('div')
+      }
+      setup(props)
+      expect(screen.queryByRole('button')).toBeTruthy()
```

Comment:
We generally prefer to use `screen.getByTestId()` to makes tests easier to understand